### PR TITLE
Introducing aws-s3-scripts

### PR DESCRIPTION
**Pull request by ZaxYonatan**
1. Script to upload a new tar to `s3://wix-code-modules`
`aws s3 cp tars-to-upload/cloud-npm-downloader-18.18.0.0.tar.gz s3://wix-code-modules/cloud-npm-downloader-18.18.0.0.tar.gz`

2. Script to override the `ga_tags/cloud-npm-downloader-18.json`
command is : 
`aws s3 cp ./cloud-npm-downloader-18.json s3://wix-code-modules/ga_tags/cloud-npm-downloader-18.json`

### CODE REVIEW

File: aws-s3-scripts/upload-tar-script.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+NODE_VERSION=$(cat ./NODE_VERSION)
+PACK_VERSION=$(cat ./PACK_VERSION)
+ARTIFACT_NAME=cloud-npm-downloader-$NODE_VERSION
+TAR_NAME=$ARTIFACT_NAME.$PACK_VERSION.tar.gz
+
+
+TAR_TO_UPLOAD=tars-to-upload/$TAR_NAME
```
Comment (Galcarmi) [inline aws-s3-scripts/upload-tar-script.sh:7]: maybe change to LOCAL_TAR_DESTINATION

File: aws-s3-scripts/upload-tar-script.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+NODE_VERSION=$(cat ./NODE_VERSION)
+PACK_VERSION=$(cat ./PACK_VERSION)
+ARTIFACT_NAME=cloud-npm-downloader-$NODE_VERSION
+TAR_NAME=$ARTIFACT_NAME.$PACK_VERSION.tar.gz
```
Comment (Galcarmi) [inline aws-s3-scripts/upload-tar-script.sh:4]: maybe NPM_TAR_NAME?

File: aws-s3-scripts/upload-tar-script.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+
+TAR_TO_UPLOAD=tars-to-upload/$TAR_NAME
+
+S3_BUCKET_PATH=s3://wix-code-modules
+S3_DESTINATION=$S3_BUCKET_PATH/$TAR_NAME
+
+# aws s3 cp tars-to-upload/cloud-npm-downloader-18.18.0.0.tar.gz s3://wix-code-modules/cloud-npm-downloader-18.18.0.0.tar.gz
+COMMAND="aws s3 cp $TAR_TO_UPLOAD $S3_DESTINATION"
```
Comment (Galcarmi) [inline aws-s3-scripts/upload-tar-script.sh:13]: UPLOAD_TAR_COMMAND?

File: aws-s3-scripts/upload-tar-script.sh [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+echo "Running the following command:\n$COMMAND"
```
Comment (Galcarmi) [inline aws-s3-scripts/upload-tar-script.sh:15]: Uploading npm tar to s3... $UPLOAD_TAR_COMMAND?

File: aws-s3-scripts/override-node-18-ga-tag.sh [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+NODE_18_GA_TAG_FILE_NAME=$ARTIFACT_NAME.json
+GA_TAGS_FOLDER_NAME="ga_tags"
+mkdir -p $GA_TAGS_FOLDER_NAME
+GA_TAG_FILE_DESTINATION="$GA_TAGS_FOLDER_NAME/$NODE_18_GA_TAG_FILE_NAME"
+
+GA_TAG_CONTENT="
+{\n
+  \t\"package\": \"$ARTIFACT_NAME\",\n
+  \t\"node-$NODE_VERSION\": \"$PACK_VERSION\"\n
```
Comment (Galcarmi) [inline aws-s3-scripts/override-node-18-ga-tag.sh:13]: i couldn't find it in Guy's script, but we need to make sure his script is aware of this new property `node-$NODE_VERSION` (node-18), I assume that in other ga-tags there are another property/properties that he's aware of

File: aws-s3-scripts/override-node-18-ga-tag.sh [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+}\n
+"
+
+echo $GA_TAG_CONTENT > $GA_TAG_FILE_DESTINATION
+
+S3_BUCKET_PATH=s3://wix-code-modules
+S3_DESTINATION=$S3_BUCKET_PATH/$GA_TAGS_FOLDER_NAME/$NODE_18_GA_TAG_FILE_NAME
+
+# aws s3 cp ga_tags/cloud-npm-downloader-18.json s3://wix-code-modules/ga_tags/cloud-npm-downloader-18.json
+COMMAND="aws s3 cp $GA_TAG_FILE_DESTINATION $S3_DESTINATION"
```
Comment (Galcarmi) [inline aws-s3-scripts/override-node-18-ga-tag.sh:23]: UPLOAD_GA_TAGS_FILE_COMMAND?

File: aws-s3-scripts/override-node-18-ga-tag.sh [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+echo "Running the following command:\n$COMMAND"
```
Comment (Galcarmi) [inline aws-s3-scripts/override-node-18-ga-tag.sh:25]: uploading ga tags file to s3... $COMMAND..

**Review by Galcarmi — COMMENTED** [review]
Hey LGTM, had some questions and mostly had naming suggestions,
feel free to ignore if you think its irrelevant

File: aws-s3-scripts/override-node-18-ga-tag.sh (hunk 7 above)
Comment (ZaxYonatan) [reply aws-s3-scripts/override-node-18-ga-tag.sh]: [renamed to UPLOAD_GA_TAGS_FILE_COMMAND](https://github.com/ZaxYonatan/node-upgrade/pull/1/commits/028e0d9d2c426365aeb381b20f52b660911bb063)

File: aws-s3-scripts/override-node-18-ga-tag.sh (hunk 6 above)
Comment (ZaxYonatan) [reply aws-s3-scripts/override-node-18-ga-tag.sh]: [renamed to UPLOAD_GA_TAGS_FILE_COMMAND](https://github.com/ZaxYonatan/node-upgrade/pull/1/commits/028e0d9d2c426365aeb381b20f52b660911bb063)

File: aws-s3-scripts/upload-tar-script.sh (hunk 4 above)
Comment (ZaxYonatan) [reply aws-s3-scripts/upload-tar-script.sh]: [renamed to UPLOAD_TAR_COMMAND](https://github.com/ZaxYonatan/node-upgrade/pull/1/commits/8ac239645658d668a4eadda029b3eafe709dfd18)

File: aws-s3-scripts/upload-tar-script.sh (hunk 3 above)
Comment (ZaxYonatan) [reply aws-s3-scripts/upload-tar-script.sh]: [renamed to UPLOAD_TAR_COMMAND](https://github.com/ZaxYonatan/node-upgrade/pull/1/commits/8ac239645658d668a4eadda029b3eafe709dfd18)

File: aws-s3-scripts/upload-tar-script.sh (hunk 2 above)
Comment (ZaxYonatan) [reply aws-s3-scripts/upload-tar-script.sh]: [renamed to NPM_TAR_NAME](https://github.com/ZaxYonatan/node-upgrade/pull/1/commits/b7f964a947ebcbc904a5f5106aebcda366d88951)

File: aws-s3-scripts/upload-tar-script.sh (hunk 1 above)
Comment (ZaxYonatan) [reply aws-s3-scripts/upload-tar-script.sh]: [renamed to LOCAL_TAR_DESTINATION](https://github.com/ZaxYonatan/node-upgrade/pull/1/commits/ad4acff5381d1742c86e7a356b659b8eb664e6de)

File: aws-s3-scripts/override-node-18-ga-tag.sh (hunk 5 above)
Comment (Galcarmi) [reply aws-s3-scripts/override-node-18-ga-tag.sh]: Oh nice

**Pull request merged**